src/IMangrove.sol

```diff
@@ -62,118 +101,334 @@ interface IMangrove is HasMgvEvents {
+  ///@param taker The taker used for transfers (should be able to deliver token amounts).
+  ///@return successes The number of successfully cleaned offers.
+  ///@return bounty The total bounty received by the caller.
+  ///@dev If an offer succeeds, the execution of that offer is reverted, it stays in the book, and no bounty is paid; The `cleanByImpersonation` function itself will not revert.
+  ///@dev Note that Mangrove won't attempt to execute an offer if the values in a target don't match its offer. To distinguish between a non-executed clean and a fail clean (due to the offer itself not failing), you must inspect the log (see `MgvLib.sol`) or check the received bounty.
+  ///@dev Any `taker` can be impersonated when cleaning because:
+  ///@dev - The function reverts if the offer succeeds, reverting any token transfers.
+  ///@dev - After a `clean` where the offer has failed, all ERC20 token transfers have also been reverted -- but the sender will still have received the bounty of the failing offers. */
   function cleanByImpersonation(OLKey memory olKey, MgvLib.CleanTarget[] calldata targets, address taker)
     external
     returns (uint successes, uint bounty);
 
   // # Maker functions
 
-  receive() external payable;
-
+  ///@notice Adds funds to Mangrove for the caller (the maker) to use for provisioning offers.
   function fund() external payable;
 
+  ///@notice Adds funds to Mangrove for the caller (the maker) to use for provisioning offers.
+  receive() external payable;
+
+  ///@notice Adds funds to Mangrove for the maker to use for provisioning offers.
+  ///@param maker The maker to add funds for.
   function fund(address maker) external payable;
 
+  ///@notice Withdraws the caller's (the maker's) native token (funds for provisioning offers) by transferring them to the caller.
+  ///@param amount the amount to withdraw.
+  ///@return noRevert whether the transfer succeeded.
   function withdraw(uint amount) external returns (bool noRevert);
 
+  ///@notice Gets the maker's balance of native tokens (funds for provisioning offers).
+  ///@param maker The maker to get the balance for.
+  ///@return balance The maker's balance of native tokens (funds for provisioning offers).
   function balanceOf(address maker) external view returns (uint balance);
 
-  function newOfferByVolume(OLKey memory olKey, uint wants, uint gives, uint gasreq, uint gasprice)
+  ///@notice Creates a new offer on Mangrove, where the caller is the maker. The maker can implement the `IMaker` interface to be called during offer execution.
+  ///@param olKey The offer list key given by (maker) `outbound`, (maker) `inbound`, and `tickSpacing`.
+  ///@param tick The tick (which is a power of 1.0001 and induces a price). Note that if `tickSpacing > 1`, the actual tick of the offer may differ from the `tick` argument by up to `tickSpacing-1`.
```

Comment:
> which is a power of 1.0001

?

Reply:
c.f. MgvOfferMaking. But yes the formulation is a bit weird.

Reply:
Not just weird, wrong no?

@adhusson

Reply:
it is supposed to be used as a power of 1.0001 😄

Reply:
lol, that's not how I read it at all 😂

Reply:
Let me propose somthing. If GitHub will allow me to make suggestions